### PR TITLE
Foundation upgrade

### PR DESCRIPTION
**Pull request by vladimirbazhanov**
# Summary

ZURB Foundation updated to latest version
jQuery updated to version 3.x

### CODE REVIEW

File: app/assets/stylesheets/browserslist [hunk 1]
```diff
@@ -0,0 +1,4 @@
+last 2 versions
```
Comment (vladimirbazhanov) [inline app/assets/stylesheets/browserslist:1]: This file was created here by foundation:install script, I dont' know why it should be here.

Comment (timurvafin) [reply app/assets/stylesheets/browserslist:1]: Please ask #dev-frontent in Slack

Comment (vladimirbazhanov) [reply app/assets/stylesheets/browserslist:1]: ok

Comment (Nikita-Polyakov) [reply app/assets/stylesheets/browserslist:1]: I think you can delete this file, because in root directory already exists `.browserlistrc` file

**Pull request merged**